EventsExpress/ClientApp/src/components/login/login.js

```diff
@@ -9,22 +9,21 @@ import ErrorMessages from '../shared/errorMessage';
 import { connect } from 'react-redux';
 import { Redirect } from 'react-router';
 import { renderTextField } from '../helpers/form-helpers';
+import { emailField } from '../helpers/validators/email-field-validator';
 
 const validate = values => {
-    const errors = {}
+    let errors = {}
     const requiredFields = [
-        'email',
         'password',
     ]
     requiredFields.forEach(field => {
```

Comment:
let move this pattern into a separate function, which would receive the whole values obj and string array of required field names.